src/components/ActionLinkList/ActionLinkList.styles.ts

```diff
@@ -0,0 +1,15 @@
+/*! Copyright (c) Microsoft Corporation. All rights reserved. Licensed under the MIT license. */
+import { mergeStyleSets } from 'office-ui-fabric-react/lib/Styling';
+import { memoizeFunction } from 'office-ui-fabric-react/lib/Utilities';
+
+export const getClassNames = memoizeFunction(() => {
+  return mergeStyleSets({
+    root: {
+      selectors: {
+        '.y-actionLink': {
```

Comment:
Curious what you think of this idea... it's a bit different than what we've been doing, but where I'm at right now, I'd prefer to avoid styling our own class names, since I think it's a bit more brittle than a styles prop, or using mergeStyles.  It's not too big of a deal, but if we rename the actionLink class we wouldn't find out that something is broken here until the visual regression tests.

I'd like it we could start a pattern of using a styles prop for stuff like this like fabric has. I think it would be better than adding a mergeStyle here, because it would lead to fewer class/styles in total.

It might be too much for this PR, but thought I'd bring it up to get feedback on the idea. 

So the ideal in my mind right now would be if we could do something like: 

```
<LayoutListItem styles={{root: { padding: '0.2rem 0 0.6rem' } }} ...
```
(with the styles coming from a function in the .styles file)

Reply:
i do like this idea within YamUI, but i'd prefer not to expose this externally unless we run into some crazy use case (since YamUI tries to alleviate this type of style overrides altogether). let's continue to think about this